test/HubPool.RelayerDispute.ts

```diff
@@ -1,91 +1,35 @@
 import { expect } from "chai";
```

Comment:
nothing changed here, just moved and cut down on redundant imports/method calls.